src/app.py

```diff
@@ -228,18 +228,21 @@ async def create_tick():
 
 
 @app.get("/")
-async def get(tick_number: int) -> Dict[str, Union[int, str, List[List[str]]]]:
+async def get(tick_number: int, player: str = -1) -> Dict[str, Union[int, str, List[List[str]]]]:
     logger.debug(f"Starting to get state for tick number {tick_number}")
     state.available_ticks_without_get = int(config["REQUESTS"]["DEFAULT_TICKS_WITHOUT_GET"])
-    # if player has no scene or player tick number is incorrect
-    # TODO remove tick number and use scene hash instead e.g. if player_scene_hash not in self.states.keys():
     current_time_ms = int(time.time() * 1000)
+
+    if tick_number == 0:
```

Comment:
Shouldn't it be -1? As first thing which Godot sends is get wiith -1 tick. But then we need to check if scene was already created.